cmd/snap-device-helper/snap-device-helper.c

```diff
@@ -160,10 +160,12 @@ int snap_device_helper_run(const struct sdh_invocation *inv) {
     if (strlen(devpath) <= strlen("/devices/")) {
         die("no or malformed devpath \"%s\"", devpath);
     }
-    if (sc_streq(action, "add") || sc_streq(action, "change")) {
+    if (sc_streq(action, "bind") || sc_streq(action, "add") || sc_streq(action, "change")) {
         allow = true;
     } else if (sc_streq(action, "remove")) {
         allow = false;
+    } else if (sc_streq(action, "unbind")) {
```

Comment:
this probably should have a comment about expecting a remove later that will be then acted on